lib/chibi/test.sld

```diff
@@ -16,8 +16,9 @@
    current-test-epsilon current-test-comparator
    current-test-filters current-test-removers
    current-test-group-filters current-test-group-removers
-   current-column-width)
+   current-column-width current-group-indent)
   (import (scheme base)
+          (scheme case-lambda)
```

Comment:
You want to remove this case-lambda and unconditionally import (chibi optional).  It has a portable implementation on snow-fort.

Reply:
implemented